Android/SortableTableView/nuget/Xamarin.SortableTableView.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Xamarin.SortableTableView</id>
+    <title>SortableTableView for Xamarin.Android</title>
+    <version>2.2.1</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>A simple TableView and an advanced SortableTableView providing a lot of customisation possibilities to fit all needs.</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <projectUrl>https://go.microsoft.com/fwlink/?linkid=871869</projectUrl>
+    <licenseUrl>https://go.microsoft.com/fwlink/?linkid=871806</licenseUrl>
+  </metadata>
+  <files>
+    <file src="output/SortableTableView.dll" target="lib/Xamarin.Android" />
```

Comment:
This should be `lib/MonoAndroid10` and `10` might not be correct, it should be the minimum version of Android that this requires to run, so if it was Android 4.4, you'd put `lib/MonoAndroid44`